templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h3>Are you sure, you want to delete driver "{{ driver.username }}"?</h3>
+  <form action="" method="post">
+    {% csrf_token %}
+
+    <input type="submit" value="Yes" class="btn btn-danger">
+     <a href="{% if object.pk %}{% url 'taxi:driver-detail' pk=object.pk %}{% else %}{% url 'taxi:driver_list' %}{% endif %}" class="btn btn-secondary">Cancel</a>
```

Comment:
According to the checklist, you should add a Cancel button which leads to the previous page the user was on. You did it correctly using the 'a' tag and href attribute with a conditional URL. Good job!